pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
+It's possible to set the bot to use a single channel for all cogs, but that will cause extreme spam and will be difficult to work with.
+
+You can start your own server and set up channels as you see fit, but for your convenience we have a template for a development server you can use: [https://discord.new/zmHtscpYN9E3](https://discord.new/zmHtscpYN9E3). Keep in mind that this is not a mirror of the Python server, and is a reduced version for testing purposes. A lot of the channels in the Python server were merged.
+
+### Set Up a Bot Account
+You will need your own bot account on Discord to test your changes to the bot.
+See [here](../creating-bot-account) for help with setting up a bot account, and invite your bot to the test server you created in the previous section.
+
+#### Privileged Intents
+
+It is necessary to explicitly request that your Discord bot receives certain gateway events.
 The Python bot requires the `Server Member Intent` to function.
 In order to enable it, visit the [Developer Portal](https://discord.com/developers/applications/) (from where you copied your bot's login token) and scroll down to the `Privileged Gateway Intents` section.
 The `Presence Intent` is not necessary and can be left disabled.
 
 If your bot fails to start with a `PrivilegedIntentsRequired` exception, this indicates that the required intent was not enabled.
 
-### Server Setup
+### Configure the Bot
+You now have both the bot's code, and a server to run it on. Now you need to connect the two by changing the bot's configurations.
 
-Setup categories, channels, emojis, roles, and webhooks in your server. To see what needs to be added, please refer to the following sections in the `config-default.yml` file:
+#### config.yml
+Entering the directory of the cloned code, you will find a file named `config-default.yml`.
+This file contains the various configurations we use to make the bot run on the Python Discord server, such as channel and role ID's, and the emojis it works with.
+It also contains configurations such as how long it takes for a help channel to time out, and how many messages a user needs to voice-verify.
 
-* `style.emojis`
-* `guild.categories`
-* `guild.channels`
-* `guild.roles`
-* `guild.webhooks`
+To run the bot in your test server, you will need to override some of those configurations.
+Open a new file in the directory called `config.yml`. The bot will first look at the items in this file, and will fall back to `config-default.yml` only if necessary.
 
-We understand this is tedious and are working on a better solution for setting up test servers.
-In the meantime, [here](https://discord.new/zmHtscpYN9E3) is a template for you to use.<br>
+For example, `config-default.yml` contains the item:
+```yaml
+bot:
+    prefix:         "!"
+```
+If you add the following lines to `config.yml`:
+```yaml
+bot:
+    prefix:         "~"
+```
+The bot will use `~` for its prefix instead.
```

Comment:
I'm not sure how necessary this example is. I think it's enough to say that "if something is present on the config.yml, it will override what is in the default-config.yml for that entry". 

This example just separates the two options of:
1. Create new file, copy and paste the provided template
2. Duplicate default-config.yml and change values

so upon reading it seems like there's only 1 option.

Reply:
Shifted things around, is that clearer?